js/src/views/Dapps/Summary/summary.js

```diff
@@ -38,7 +38,14 @@ export default class Summary extends Component {
       return null;
     }
 
-    const url = `/app/${app.builtin ? 'global' : 'local'}/${app.url || app.id}`;
+    let type = 'builtin';
```

Comment:
Erk. Actually missing the built-in type determination, this and next comment. Silly me, only tested GAVcoin, half-assed job - this is a bug. Good catch.